acceptance/test.toml

```diff
@@ -13,3 +13,13 @@ TimeoutCloud = '25m'
 
 Env.PYTHONDONTWRITEBYTECODE = "1"
 Env.PYTHONUNBUFFERED = "1"
+EnvRepl.PYTHONUNBUFFERED = false
+
+# On Windows, MSYS2 automatically transforms absolute paths (e.g., /Users/$username/$UNIQUE_NAME)
+# into Windows-style paths (C:/Program Files/Git/Users/$username/UNIQUE_NAME) before CLI execution.
+# Setting MSYS_NO_PATHCONV=1 prevents this automatic path conversion.
+# This is appropriate for our test suite because:
+# 1. Absolute paths typically reference files within Databricks workspaces, not local filesystem paths
+# 2. For local file references, our tests predominantly use relative paths
+Env.MSYS_NO_PATHCONV = "1"
+EnvRepl.MSYS_NO_PATHCONV = false
```

Comment:
not necessary anymore, this will be the default since https://github.com/databricks/cli/pull/2868

Reply:
Thanks, updated.